### DISCUSSION TITLE
Can’t Restore Purchase

### DISCUSSION THREAD

**Discussion by cansurmeli** [blinksh/blink#1718] [Support] [answered]
Hello.

I currently have an on-going subscription for Blink which I can see in my App Store page as well. I’m setting up a new iPhone where I installed Blink beta (v16.1.4 (717)).

Everything seems to work fine except that I can’t seem to restore my purchase. If I click on the button under `config`, some progress seems to happen that the wheel turns but afterwards nothing changes nothing gets restored.

Tried it many times. Also tried killing Blink completely or restarting my iPhone. Nothing changes.

Thanks

**Comment by carloscabanero** [comment, marked as answer]
Hi! So Blink Beta does not require a subscription except for the Blink Build services. If you have signed up for those, and want to access Build from your phone, install the AppStore version and restore the purchase there. Once that version has access to the Build information, then install the Beta again.

**Reply by cansurmeli** [reply]
Hey Carlos. Got it. Thanks a bunch!
